cellxgene_resources/cellxgene_mods.py

```diff
@@ -951,33 +952,30 @@ def evaluate_var_df(adata):
         report('Features in var.index are gene symbols and/or contain deprecated Ensembl IDs', 'ERROR')
         return
     valid = True
-    obs_organisms = adata.obs['organism_ontology_term_id'].unique().tolist()
+    uns_organism = adata.uns['organism_ontology_term_id']
     var_organisms = [o.value for o in var_organism_objs]
 
     if 'NCBITaxon:2697049' in var_organisms:
         report('There are covid genes present in var')
         var_organisms.remove('NCBITaxon:2697049')
-    if 'NCBITaxon:2697049' in obs_organisms:
-        report('Covid is found in obs metadata', 'ERROR')
-        obs_organisms.remove('NCBITaxon:2697049')
+    if 'NCBITaxon:2697049' == uns_organism:
+        report('Covid is found in uns metadata', 'ERROR')
+        uns_organism = None
         valid=False
-    if len(obs_organisms) > 1:
-        report(f'Multiple organisms found in obs metadata: {obs_organisms}', 'ERROR')
-        valid = False
     if len(var_organisms) > 1:
         report(f'Multiple organisms found in var index: {var_organisms}', 'ERROR')
         valid = False
     if valid:
-        if obs_organisms[0] == var_organisms[0]:
+        if uns_organism == var_organisms[0]:
             report(f'Single organism found: {var_organisms}', 'GOOD')
         else:
-            report(f'Different organisms found between var index and obs metadata: {var_organisms[0]}, {obs_organisms[0]}', 'ERROR')
+            report(f'Different organisms found between var index and uns metadata: {var_organisms[0]}, {uns_organism}', 'ERROR')
             return
     else:
         return
 
     # Check the number of genes threshold base on biotype per specific organism
-    organism = obs_organisms[0]
+    organism = uns_organism
     org_obj = [i for i in gencode.SupportedOrganisms if i.value==organism][0]
```

Comment:
I would just simplify this - remove the `organism` definition & use i.value==uns_organism